lib/constants/styles.dart

```diff
@@ -69,6 +69,14 @@ class ButtonStyles {
       ),
     );
   }
+
+  static ButtonStyle homePageButton(BuildContext context) {
+    return ButtonStyle(
+        shape: MaterialStateProperty.all<RoundedRectangleBorder>(
+            RoundedRectangleBorder(
+      borderRadius: BorderRadius.circular(Radii.buttonRadius),
+    )));
+  }
```

Comment:
You can extend the ElevatedButton style instead of defining a ButtonStyle using the constructor. This is how it would look:

```
static ButtonStyle homePageButton(BuildContext context) {
    return ElevatedButton.styleFrom(
        shape: RoundedRectangleBorder(
      borderRadius: BorderRadius.circular(Radii.buttonRadius),
    ));
  }
```

In this case both look the same, but using this approach is generally better because it extends the ElevatedButton style instead of replacing it with a new one.

Reply:
Gotcha, changed to extending ElevatedButton style